@ornikar/eslint-config/tests/prefer-modern-math-apis.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const bar = 0;
+
+// Incorrect uses
+
+// eslint-disable-next-line no-unused-expressions, unicorn/prefer-modern-math-apis
+Math.log(bar) * Math.LOG10E;
```

Comment:
```suggestion
export foo1 = Math.log(bar) * Math.LOG10E;
```

ca evite tous les disable